Tests/test_codecs.py

```diff
@@ -9,11 +9,14 @@
 import unittest
 import codecs
 
-import System.Text
+from iptest import run_test, is_ironpython
```

Comment:
I think `is_cli` is used throughout the tests. Might be better to use it here for consistency. Would be nice to eliminate one of them at some point.

Reply:
Most tests use `is_cli`, a few tests use both or only `is_ironpython` (`test_class`, `test_excinfo`, `test-exec`). It seems that originally `is_ironpython` was defined as `is_silverlight or is_cli`. With the drop of silverlight, they are now equivalent. It is a good idea to eliminate one of them. I am OK either way: `is_ironpython` is more informative, `is_cli` is shorter. It will be less work to settle on `is_cli` as the changes will be limited. If you are OK with that, I can include those changes in this commit.

Reply:
Ah that explains it. Yes I'm fine with removing `is_ironpython` in this PR. Eventually it might be nice to rename `is_cli` to `is_ironpython` but I'd wait until we diverge more from or stop supporting ipy2.